components/ButtonLink.vue

```diff
@@ -23,7 +23,7 @@ export default {
     props: {
         href: {
             type: String,
-            default: "#0",
+            default: "#",
```

Comment:
I'm not familiar enough with Vue to understand the purpose of this change. Is this just the default empty hash? Was this just a typo before?

Reply:
Oh I thought this was a typo, but after a bit research I see that the `0` does actually provide some functionality. I'll revert this. Thanks!